Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -2,28 +2,62 @@
 using System.Collections.Generic;
 using System.Linq;
 using Neuro.Pathfinding.DataStructures;
+using Reactor.Utilities.Extensions;
 using UnityEngine;
 
 namespace Neuro.Pathfinding;
 
 public sealed class PathfindingHandler
 {
-    private const float GRID_DENSITY = 6f; // TODO: Fine-tune individual maps to optimize performance
-    private const int GRID_BASE_WIDTH = 100;
+    // TODO: Reset these in an un-initialize function
+    internal float gridDensity = 0;
+    internal int gridBaseWidth = 0;
 
-    private const int GRID_SIZE = (int)(GRID_BASE_WIDTH * GRID_DENSITY);
-    private const int GRID_LOWER_BOUNDS = GRID_SIZE / -2;
-    private const int GRID_UPPER_BOUNDS = GRID_SIZE / 2;
+    internal int gridSize = 0;
+    private int _gridLowerBounds = 0;
+    private int _gridUpperBounds = 0;
 
-    private Node[,] grid;
+    private Node[,] _grid;
+    private GameObject _nodeVisualPointParent;
+    private GameObject _taskVisualPointParent;
+    private GameObject _pathToTask;
 
```

Comment:
This whole section is inaccessible now
![image](https://user-images.githubusercontent.com/32238504/230753302-2c2a5996-51ee-4736-aeb0-18ad7cab6fb8.png)

Reply:
You need currently to restart the game between different maps. When #34 gets merged we can add an `OnDestroy()` method to reset the node grid's density

Reply:
What values are not being reset?

Reply:
`gridDensity` and `gridBaseWidth` need to be restored to `0` between maps.

Reply:
Why do you even check if they are 0 before resetting them? Initialize is only called when a new map is loaded or when manually triggered from the debug window

Reply:
If I just set the value then it is impossible to manually alter the values in the debug window. If you think that is unnecessary or could be done better then I am willing to change it.

Reply:
I have changed how overriding works which solves the problem as long as the override box in the debug menu is not checked between maps.